src/hotspot/share/utilities/globalDefinitions_xlc.hpp

```diff
@@ -75,11 +75,11 @@
 // NULL vs NULL_WORD:
 // Some platform/tool-chain combinations can't assign NULL to an integer
 // type so we define NULL_WORD to use in those contexts. For xlc they are the same.
-#define NULL_WORD  NULL
+#define NULL_WORD  nullptr
```

Comment:
Hmmm so maybe NULL_WORD can be retired after this change?

Reply:
NULL_WORD is mostly used in the x86 assembler, for overloading purposes.  I've made a couple of attempts
at improving it but it was messy because of NULL and the lack of a known type for it (though decltype could
be used for that now).  It might be easier with NULL => nullptr everywhere.

There are a small number of uses of NULL_WORD elsewhere, most/all of which I think are mistakes.